app/views/DR/HomeScreen/index.js

```diff
@@ -164,6 +165,46 @@ class HomeScreen extends Component {
       state: { confirmed, deaths, recovered, current, refreshing, notified },
     } = this;
 
+    const dashboardCards = () => {
+      const dashboardCategories = [
+        { label: 'label.positive_label', styles: '', value: confirmed },
+        {
+          label: 'label.deceased_label',
+          styles: { color: Colors.BUTTON_LIGHT_TEX },
+          value: deaths,
+        },
+        {
+          label: 'label.recovered_label',
+          styles: { color: Colors.GREEN },
+          value: recovered,
+        },
+        {
+          label: 'label.case_day_label',
+          styles: { color: Colors.SUN },
+          value: current,
+        },
+      ];
+      const CONFIRMED_COVID_CASES_URL =
+        'https://coronavirusrd.gob.do/casos-de-covid-19-confirmados/';
+
+      return dashboardCategories.map(obj => {
+        return (
```

Comment:
You do not need this return, just delete it and remove `{}`
```suggestion
      return dashboardCategories.map(obj => 
         (
```